test/commands/auth/token.js

```diff
@@ -9,14 +9,15 @@ const cli = proxyquire('heroku-cli-util', {'netrc-parser': netrc})
 
 const cmd = proxyquire('../../../src/commands/auth/token', {'heroku-cli-util': cli})
 const expect = require('unexpected')
+const validator = require('validator')
 
 describe('auth:token', () => {
   beforeEach(() => cli.mockConsole())
 
   it('shows logged in user', () => {
     netrc.prototype.machines = {'api.heroku.com': {password: 'myapikey'}}
     return cmd.run({})
-      .then(() => expect(cli.stdout, 'to equal', 'myapikey\n'))
+      .then(() => expect(validator.isUUID(cli.stdout.trim()), 'to be ok'))
```

Comment:
There is definitely something interesting going on here after updating to v8.0.0. It appears to be giving back your own session key.

Not sure if this should be dealt with there though, or in a separate PR.